.env.template

```diff
@@ -1,7 +1,13 @@
-MYSQL_ROOT_PASSWORD=password!
-LOCAL_DEV_DOMAIN=docker.local
-WP_SITE_TITLE="Site Title"
+#Domain configuration variables
+LOCAL_DEV_DOMAIN=jam3site.local
+
+#Wordpress configuration variables
+MYSQL_ROOT_PASSWORD=H3b3Llk22
+WP_SITE_TITLE="Jam3 Site"
 WP_ADMIN_USER=admin
-WP_ADMIN_PASSWORD=password!
-WP_ADMIN_EMAIL=email@jam3.com
+WP_ADMIN_PASSWORD=H3b3Llk22
+WP_ADMIN_EMAIL=rodrigo.techera@jam3.com
```

Comment:
Remove your credentials from here, and we will squash this PR and delete the branch